docs/blog/2025-q2-changelog.mdx

```diff
@@ -0,0 +1,56 @@
+---
+slug: 2025-q2-updates-and-changes
+title: 2025 Q2 Updates and Changes
+---
+
+## Meltano
+
+We've released multiple versions of Meltano during Q2, from [3.7.2](https://github.com/meltano/meltano/releases/tag/v3.7.2) through [3.7.9](https://github.com/meltano/meltano/releases/tag/v3.7.9)!
+
+These releases include, among other things, the following changes:
+
+* [New unified `--state-strategy` CLI option](https://github.com/meltano/meltano/pull/9329) that provides clearer control over state handling in `run`, `el`, and `elt` commands, replacing the legacy `--merge-state` flag.
+* [Plugin type inference for `meltano add`](https://github.com/meltano/meltano/pull/9317) - users no longer need to specify plugin types explicitly. Commands like `meltano add tap-github` now work automatically.
+* [Automatic catalog refresh with `--full-refresh`](https://github.com/meltano/meltano/pull/9335) ensures data schema changes are properly detected when performing full refreshes.
+* [Local timezone support in logging](https://github.com/meltano/meltano/pull/9328) - new `utc` parameter in logging formatters allows timestamps to be displayed in local time.
+* [JSON output for `meltano select`](https://github.com/meltano/meltano/pull/9266) with the new `--json` flag provides machine-readable output for selection patterns.
```

Comment:
These haven't been released yet and are instead part of the v3.8 milestone, which I plan to publish in a few days. We can still decide to include them here though.

Reply:
I just did a one-liner here to say we did bug fixes and under the hood improvements.